Sming/Arch/Esp32/Tools/install.sh

```diff
@@ -31,18 +31,20 @@ $PKG_INSTALL ${PACKAGES[*]}
 if [ -d "$IDF_PATH" ]; then
     printf "\n\n** Skipping ESP-IDF clone: '$IDF_PATH' exists\n\n"
 else
-    git clone -b release/v4.1 https://github.com/espressif/esp-idf.git $IDF_PATH
+    git clone -b release/v4.3 https://github.com/espressif/esp-idf.git "$IDF_PATH"
```

Comment:
@mikee47 How about cloning the content to `/opt/esp-idf-X` where X is the current version. Example `/opt/esp-idf-4.3` and after that making a symbolic link to `/opt/esp-idf` ?

Reply:
Good idea. I'll take a look at that, should simplify things as removal of v4.1 then becomes optional :-)
The toolchains aren't compatible either, so v4.3 needs the newer toolchains and v4,1 won't build with those. However, I've just spotted a `toolchain_versions.mk` file in the idf directory though so can use that to ensure the correct tools are used.